### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by butcherZ**

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # .env
 PORT_SERVER = 8000
 FRONT_URL = http://localhost:3000
-MY_EMAIL = daydreamnation.y@gmail.com
\ No newline at end of file
+MY_EMAIL = hypertubes.prod@gmail.com
+MONGODB_URI = mongodb+srv://zyuan:hybertube@hybertube-umhyd.mongodb.net/hybertube?retryWrites=true
```
Comment (butcherZ) [inline backend/.env]: the uri has been changed to this one
`mongodb+srv://zyuan:hypertube@hypertubedb-umhyd.mongodb.net/hypertube?retryWrites=true&w=majority`

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+services:
+
+  backend:
+    container_name: backend
+    restart: always
+    image: node:12.2.0-alpine
```
Comment (butcherZ) [inline docker-compose.yml]: - [ ] should change node version to 11.0.0 to support big-num

- [ ] should add ffmpeg image for converting video

File: backend/config/keys.js [hunk 3]
```diff
@@ -15,14 +15,5 @@ module.exports = {
   github: {
```
Comment (butcherZ) [inline backend/config/keys.js]: should I put the clientID and secret to env as well?

File: docker-compose.yml (hunk 2 above)
Comment (sylflo) [reply docker-compose.yml]: For the ffmpeg merge the code and rebase on your streaming branch

Comment (sylflo) [reply docker-compose.yml]: e0fe77798c42e27cda25aa568f6c845bfd6dfe0a

**Pull request merged**